.github/workflows/daily-testgrid.yaml

```diff
@@ -22,9 +23,15 @@ jobs:
     - name: tgrun-build
       run: make build
 
-    - name: testgrid-queue
+    - name: testgrid-queue-prod
       env:
         TESTGRID_API: "https://api.testgrid.kurl.sh/"
       run: |
-        ./bin/tgrun queue --ref "${GITHUB_SHA:0:7}-$(date --utc +%FT%TZ)-daily"
-      if: ${{ github.repository_owner == 'replicatedhq' }}
+        ./bin/tgrun queue --ref "PROD-daily-${GITHUB_SHA:0:7}-$(date --utc +%FT%TZ)"
```

Comment:
I'd be happy with dropping the github SHA from this

Reply:
Agreed!